base/uk.ac.stfc.isis.ibex.ui.devicescreens/src/uk/ac/stfc/isis/ibex/ui/devicescreens/ComponentIcons.java

```diff
@@ -252,6 +252,8 @@ private static String getIconNameForType(ComponentType componentType) {
             	return "function_generator";
             case PHOTON_COUNTER:
             	return "photon_counter";
+            case LITRONPOWER:
```

Comment:
i've already done this in https://github.com/ISISComputingGroup/ibex_gui/commit/74ce4845f5b7067a54dd4e45347e0a3de5035838

Reply:
no I haven't, sorry...